File: Assignment40/program1.cpp
/////////////////////////////////////////////////////////////////// 
//
// File name : program1.cpp
// Description : Pattern printing using recursion
// Author : Adinath Santosh Pawar
// Date : 21/7/25
// 
///////////////////////////////////////////////////////////////////

#include<iostream>
using namespace std;

void Display()
{
    static int i = 1;

    if(i <= 5)
    {
        cout<<"* ";
        i++;
        Display();
    }
}


int main()
{
    Display();

    return 0;
}

File: Assignment46/program1.java
import java.util.*;

/////////////////////////////////////////////////////////////////// 
//
// File name : program1.java
// Description : Pattern printing
// Author : Adinath Santosh Pawar
// Date : 20/8/25
// 
///////////////////////////////////////////////////////////////////

class Pattern
{
    public void Pattern(int iRow, int iCol)
    {
        for(int i = 1; i <= iRow; i++)
        {
            char ch = 'A';
            for(int j = 1; j <= iCol; j++)
            {
                System.out.print(ch + " ");
                ch++;
            }
            System.out.println();
        }
    }
}
class program1
{
    public static void main(String A[])
    {
       Pattern pobj = new Pattern();
       Scanner sobj = new Scanner(System.in);

       System.out.print("Enter the number of rows ");
       int iRow = sobj.nextInt();

       System.out.print("Enter the number of columns ");
       int iCol = sobj.nextInt();
       pobj.Pattern(iRow,iCol);
    }
}